etc/ci/performance/page_load_test/example/example_async.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width" />
-    <title></title>
+
```

Comment:
Rather than replacing with a blank line, it would be better to remove the line entirely (and in all other modified files).